synapse/storage/databases/main/deviceinbox.py

```diff
@@ -63,6 +64,15 @@
 logger = logging.getLogger(__name__)
 
 
+# How long to keep messages in the device federation inbox before deleting them.
+DEVICE_FEDERATION_INBOX_CLEANUP_DELAY_MS = 7 * 24 * 60 * 60 * 1000  # 7 days
```

Comment:
Probably worth building these up to avoid mistakes and make it more clear using the constants 
https://github.com/element-hq/synapse/blob/f500c7d982b8d00773a8652affa75800906a570e/synapse/util/constants.py#L15-L20

We will need a few more:
```
ONE_MINUTE_SECONDS = 60
ONE_HOUR_SECONDS = 60 * ONE_MINUTE_SECONDS
ONE_DAY_SECONDS = 24 * ONE_HOUR_SECONDS

MILLISECONDS_PER_SECOND = 1000
```

```suggestion
DEVICE_FEDERATION_INBOX_CLEANUP_DELAY_MS = 7 * ONE_DAY_SECONDS * MILLISECONDS_PER_SECOND
```

Reply:
> Are we going to remove the other constants?

Yes, we probably should, but didn't want to do it here

> How are we going to handle the seconds variation (vs MS)?

Generally we rarely operate in seconds, except at twisted boundaries. TBH I think its fine in those cases to just `/ 1000`.